mopidy/core/playback.py

```diff
@@ -207,12 +207,16 @@ def _change_track(self, tl_track, on_error_step=1):
         if old_state == PlaybackState.PLAYING:
             self._play(on_error_step=on_error_step)
         elif old_state == PlaybackState.PAUSED:
-            # NOTE: this is just a quick hack to fix #1177 as this code has
-            # already been killed in the gapless branch.
+            # NOTE: this is just a quick hack to fix #1177 and #1352 as this
+            # code has already been killed in the gapless branch.
             backend = self._get_backend()
             if backend:
-                backend.playback.prepare_change()
-                backend.playback.change_track(tl_track.track).get()
+                success = (
+                    backend.playback.prepare_change().get and
```

Comment:
Also, the default implementation of `backend.playback.prepare_change()` returns `None`, so it will always fail. I think `success` should only take the return value of `change_track()`.